src/main/java/ch/heigvd/dai/models/Pixel.java

```diff
@@ -0,0 +1,38 @@
+package ch.heigvd.dai.models;
+
+/** Defines a pixel with 3 color components */
+public class Pixel {
+  private int red;
+  private int green;
+  private int blue;
+
+  public Pixel(int red, int green, int blue) {
+    this.red = red;
+    this.green = green;
+    this.blue = blue;
+  }
+
+  public int getRed() {
+    return red;
+  }
```

Comment:
Why use int and not bytes ? Also I think it would be simpler for reading and writing to store a byte array of length 3. Then :
- getRed would look like "return colors[RED]" assuming you give the class constant values like int RED = 0, ...
- reading would look like : "bis.read(byteArrayOfLength3); image[w][height - 1 - h] = new Pixel(red, green, blue);"
- writing would look the same as reading

Reply:
After discussion, will only switch to bytes but keep the rest this way